administration/src/util/PlainDate.ts

```diff
@@ -56,11 +58,26 @@ class PlainDate {
    * @param format A custom format as used by date-fns
    * @example
    */
-  static fromCustomFormat(value: string, format: 'dd.MM.yyyy' = 'dd.MM.yyyy'): PlainDate {
+  static fromCustomFormat(value: string, format: string = 'dd.MM.yyyy'): PlainDate {
     const date = parse(value, format, new Date(0))
     return PlainDate.fromLocalDate(date)
   }
 
+  static safeFromCustomFormat(value: string | null, format: string = 'dd.MM.yyyy'): PlainDate | null {
+    if (value === null) {
+      return null
+    }
+    try {
+      return PlainDate.fromCustomFormat(value, format)
+    } catch {
+      return null
+    }
```

Comment:
```suggestion
return value && PlainDate.fromCustomFormat(value, format).catch(() => null);
```

Reply:
Sadly this does not work as
a) `value &&` might return empty strings `""`
b) `.catch()` does not exist on non-promise values